themes/src/main/resources/theme/base/admin/resources/js/services.js

```diff
@@ -312,7 +312,7 @@ module.factory('ComponentUtils', function() {
                     config[prop.name] = configProperty;
                 }
 
-                if (configProperty.length == 0 || configProperty[configProperty.length - 1].length > 0) {
+                if (configProperty.length === 0) {
```

Comment:
TBH, I'm not sure about this change. I think it was intentional to add the extra empty line, which is now not happening in some cases. I'd leave it as it was before. (We'd just need to figure out why the extra line is not displayed after clicking "Save" btn.)

In any case, if we made this change, we'd need to run all UI tests as it has a bigger impact, not just on IdPs.